docs/src/main/sphinx/release/release-427.md

```diff
@@ -0,0 +1,108 @@
+# Release 427 (25 Sep 2023)
+
+## General
+
+* Add support for IPv4 mapped addresses in `contains`. ({issue}`18497`)
+* Make plans with complex aggregations deterministic. ({issue}`19103`)
+* Improve performance of `GROUP BY`, `DISTINCT`, and other hashing operations. ({issue}`19059`)
+* Fix query failure due to exceeding node memory limits with `INSERT`
+  statements. ({issue}`18771`)
+* Fix incorrectly reporting `ns` instead of `ms` in query plans. ({issue}`19047`)
+
+## Security
+
+* Add access control check to redirected tables when accessing table comments. ({issue}`18514`)
+
+## Delta Lake connector
+
+* Add support for reading tables with deletion vectors. ({issue}`16903`)
+* Add support for writer
+  [version 7](https://docs.delta.io/latest/versioning.html#features-by-protocol-version). ({issue}`15873`)
+* Add support for writing the `timestamp` type. ({issue}`16927`)
+* Reduce data read from Parquet files for queries with filters. ({issue}`19032`)
+* Improve performance of writing to Parquet files. ({issue}`19122`)
+* Fix error reading Delta Lake table history when the initial transaction logs
+  have been removed. ({issue}`18845`)
+
+## Elasticsearch connector
+
+* Fix query failure when a `LIKE` clause contains multi-byte characters. ({issue}`18966`)
+
+## Hive connector
+
+* Add support for `double` to `varchar` coercion in Hive tables. ({issue}`18930`)
+* Add automatic type coercion from `varchar(0)` to `varchar(1)`. ({issue}`18811`)
```

Comment:
what. . varchar(0) is valid? That is weird

Reply:
Causes errors according to the PR adding it.